sample/0.1.0/content/entrypoint.ps1

```diff
@@ -0,0 +1,12 @@
+param(
+    [Alias('f')]
+    [Switch]$Fail
+)
+
+echo "This is a script used to test PRM"
+echo "Contents of target dir: "
+ls
+
+if ($Fail) {
+  return 1
```

Comment:
In PowerShell, we need to write an error and then explicitly exit with the desired code. We will likely want to investigate handling PowerShell-native terminating exceptions, such as those returned from `Throw "terminating message"`.

```suggestion
Write-Error "-f flag was triggered, the script will now exit with error code 1"
exit 1
```